news/_posts/2022-06-06-prestashop-feature-freeze-v8.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title:  "PrestaShop Version 8.0 Feature Freeze: Important Features"
+subtitle: "Feature Freeze: an Inside Look at Most Important Features"
+date:   2022-06-06 09:10:11
+authors: [ PrestaShop ]
+icon: icon-compass
+tags:
+ - news
+ - version
+ - 8.0
+---
+
+Dear developers and contributors,
+
+We are very pleased to be announcing the feature freeze of PrestaShop version 8.0. In this post, we will be looking at the most important features of this feature freeze, letting you know why we have frozen new features and explain what this means for the next few months.
+
+### Feature freeze is now effective
+
+Therefore, as of today, the feature freeze is now effective and no new features will be added to version 8.0, so we can work on implementing new features and working on their quality.
+
+In the next few months, we will still be welcoming new features. We will be creating the 8.0.x branch to stabilize the code of the next 8.0 and new features will be added on “develop”.
+
+### Most important features of version 8.0
+
+In this section, we wanted to highlight the most important features of this version 8.0 feature freeze, visible below:
+1. Update to[ Symfony 4.4](https://symfony.com/releases/4.4) (LTS version, support until November 23)
+2. Compatibility with[ PHP 8](https://www.php.net/) / 8.1
+3. Many different items regarding the open source project separation, removing all dependencies to the company to make the project as neutral as possible
```

Comment:
Maybe we could link to a more detailed explanation. The demo done by @ttoine maybe?

Reply:
Hi Matks, would this video be the right one? https://youtu.be/LyNUS0STl4Q?t=943 Or this one, but it's more 2020-2021 https://youtu.be/Bp8ef2jjIBw Let me know.

Reply:
I was thinking about this one https://www.youtube.com/watch?t=128&v=jwOy3gx-MNs&feature=youtu.be which is in English 🇬🇧

Reply:
```suggestion
3. Many different items regarding the [open source project separation](https://www.youtube.com/watch?t=128&v=jwOy3gx-MNs&feature=youtu.be), removing all dependencies to the company to make the project as neutral as possible
```